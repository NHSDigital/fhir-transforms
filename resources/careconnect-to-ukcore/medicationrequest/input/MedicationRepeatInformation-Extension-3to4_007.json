{"resourceType":"MedicationRequest","extension":[{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationRepeatInformation-1","extension":[{"url":"numberOfRepeatPrescriptionsAllowed","valuePositiveInt":12},{"url":"numberOfRepeatPrescriptionsIssued","valueUnsignedInt":0},{"url":"authorisationExpiryDate","valueDateTime":"2020-11-01"}]}]}
