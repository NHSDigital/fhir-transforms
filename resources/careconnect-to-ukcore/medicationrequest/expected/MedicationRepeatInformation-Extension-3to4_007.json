{
  "resourceType": "MedicationRequest",
  "extension": [
    {
      "extension": [
        {
          "url": "numberOfRepeatPrescriptionsIssued",
          "valueUnsignedInt": 0
        },
        {
          "url": "authorisationExpiryDate",
          "valueDateTime": "2020-11-01"
        }
      ],
      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-UKCore-MedicationRepeatInformation"
    }
  ],
  "dispenseRequest": {
    "numberOfRepeatsAllowed": 12
  }
}